### PR TITLE
Notes taken during today's prow meeting with James Munnelly

### PR DESCRIPTION
**Pull request by maelvls**
I tried to reproduce the steps that @munnerz described today. I think I missed many thing; fortunately, we might be automating that away soon 😇

### CODE REVIEW

File: prow/README.md [hunk 1]
```diff
@@ -1,41 +1,72 @@
...
-```
-$ bazel run //prow/cluster:production.apply
-```
+   ```diff
+   git_repository(
+       name = "test_infra",
+   -   commit = "a8cee5a60a2d9476341cf843867221a8bd18a3e8",
+   +   commit = "eca83d2ac2b48c2732aab0c90c6eff6e564d4a21",
+       remote = "https://github.com/kubernetes/test-infra.git",
+   )
```
Comment (munnerz) [inline prow/README.md:49]: This step is not necessary :) you need to change the `prow/version` file instead

Comment (munnerz) [reply prow/README.md:49]: https://github.com/jetstack/testing/blob/449fa5d671c1bf2f98ce2f3f1a5cab455b93b70b/images/builder/ci-runner.sh#L78-L84 <- this is the only place that the test_infra repository dependency is actually used

Comment (maelvls) [reply prow/README.md]: Fixed in 6ba1687! Let me know if it looks better

**Review by jakexks — APPROVED** [review]
/lgtm

**Pull request merged**